### PR TITLE
[2.26.x] Upgrade Tika to 1.26

### PR DESCRIPTION
**Pull request by pvargas**
#### What does this PR do?
This PR upgrades Tika from version 1.24 to version 1.26, in order to [addresses an inconsistency](https://issues.apache.org/jira/browse/TIKA-3318) regarding MP3 format duration units.

#### Who is reviewing it? 
@alexabird 
@jrnorth
@jlcsmith

#### How should this be tested?
Verify that the metacard for any ingested MP3 file should have the duration in seconds and not in milliseconds.

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by jlcsmith** [comment]
build now

File: catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java [hunk 1]
```diff
@@ -472,6 +472,8 @@ public void testMp3() throws Exception {
         containsString("<meta name=\"xmpDM:artist\" content=\"Test Artist\" />"));
     assertThat(metacard.getContentTypeName(), is("audio/mpeg"));
     assertThat(metacard.getAttribute(Core.DATATYPE).getValue(), is(SOUND));
+    assertThat(
```
Comment (jrnorth) [inline catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java:475]: We should add an assertion for the `media.duration` metacard attribute.

File: catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java [hunk 2]
```diff
@@ -472,6 +474,9 @@ public void testMp3() throws Exception {
...
+        metacard.getMetadata(), containsString("<meta name=\"xmpDM:duration\" content=\"2.4"));
+    assertTrue(metacard.getAttribute(Media.DURATION).getValue().toString().startsWith("2.4"));
```
Comment (jrnorth) [inline catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java:479]: If this assertion fails, all the message will say is something like "expected true but was false", which isn't helpful in diagnosing the failure.

We should prefer `assertThat()`: `assertThat(metacard.getAttribute(Media.DURATION).getValue(), is(2.4))`

See https://codice.atlassian.net/wiki/spaces/DDF/pages/6291523/Unit+Testing#UnitTesting-Assertions

**Review by jrnorth — APPROVED** [review]

**Comment by alexabird** [comment]
build now

**Pull request merged**